rules/windows/privilege_escalation_group_policy_scheduled_task.toml

```
@@ -20,28 +20,36 @@ Group Policy Objects can be used by attackers to execute scheduled tasks at scal
 given GPO. This is done by changing the contents of the `<GPOPath>\\Machine\\Preferences\\ScheduledTasks\\ScheduledTasks.xml`
```

This suggestion is really for an edit on line 19.

```suggestion
Group Policy Objects (GPOs) can be used by attackers to execute scheduled tasks at scale to compromise objects controlled by a given GPO. This is done by changing the contents of the `<GPOPath>\\Machine\\Preferences\\ScheduledTasks\\ScheduledTasks.xml````